tests/objects/test_font.py

```diff
@@ -101,3 +105,33 @@ def test_data_images_init() -> None:
     assert font.data["bbb/c"] == b"456"
     assert font.images["a.png"] == b"\x89PNG\r\n\x1a\n"
     assert font.images["b.png"] == b"\x89PNG\r\n\x1a\n"
+
+
+@pytest.mark.parametrize(
+    "lazy", [None, False, True], ids=["lazy-unset", "non-lazy", "lazy"]
+)
+def test_pickle_lazy_font(datadir: Path, lazy: Optional[bool]) -> None:
+    if lazy is not None:
+        font = Font.open(datadir / "UbuTestData.ufo", lazy=lazy)
+    else:
+        font = Font()
```

Comment:
Did you want to load a font here?

Reply:
No, here I'm testing the lazy=None case for a Font that was not read from a file but created from scratch (added a comment to clarify https://github.com/fonttools/ufoLib2/pull/230/commits/c6e188c2b2a5c4ca60394e66c89be5df9307df77)